docs/sections/user_guide/yaml/components/ungrib.rst

```diff
@@ -0,0 +1,38 @@
+.. _ungrib_yaml:
+
+ungrib
+======
+
+Structured YAML to run the WRF preprocessing component ``ungrib`` is validated by JSON Schema and requires the ``ungrib:`` block, described below. If ``ungrib`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Here is a prototype UW YAML ``ungrib:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/ungrib.yaml
+
+
+UW YAML for the ``ungrib:`` Block
+----------------------------------------
+
+execution:
+^^^^^^^^^^
+
+See :ref:`here <execution_yaml>` for details.
+
+
+gfs_file:
+^^^^^^^^^
+
+The path to the GRIB-formatted file to be processed by ``ungrib``.
```

Comment:
This is changing in #441 -- should we wait on that an update this to match to avoid a second documentation PR?

Reply:
Ah good point, let me update this relative to the function of #441 and hold the merge for its completion